scripts/tracing.js

```diff
@@ -1,41 +1,42 @@
 'use strict';
 
-const { propagation } = require('@opentelemetry/api');
+const { NodeSDK } = require('@opentelemetry/sdk-node');
```

Comment:
I'm a bit surprised this isn't just the API. I thought that consumers only needed to deal with that.

Reply:
This is part of the instrumentation and exporter configuration.